.github/actions/repo-gardening/src/get-next-valid-milestone.js

```diff
@@ -29,10 +30,12 @@ async function getNextValidMilestone( octokit, owner, repo, plugin = 'jetpack' )
 	for await ( const response of responses ) {
 		// Find a milestone which name is a version number
 		// and it's due dates is earliest in a future
-		const reg = new RegExp( plugin + '\\/\\d\\.\\d' );
+		const reg = new RegExp( '^' + plugin + '\\/\\d\\.\\d' );
```

Comment:
If you really want to keep the `\\/` I won't insist, but the plus is necessary so this doesn't break once we hit Jetpack 10.0.
```suggestion
		const reg = new RegExp( '^' + plugin + '\\/\\d+\\.\\d' );
```

Reply:
> If you really want to keep the `\\/` I won't insist

Hm, what would be the alternative? We want to match `plugin/0.0`, so we probably leave escaped `/`.

Reply:
> Hm, what would be the alternative? We want to match `plugin/0.0`, so we probably leave escaped `/`.

https://github.com/Automattic/jetpack/pull/19189#discussion_r596210722